README.md

```diff
@@ -1,59 +1,111 @@
+
 ## Requirements
 
-You must have PHP 7.1 or greater, which is a requirement of PHPStan, the static analyzer backing this tool.
+* PHP >=7.1
 
-## Usage
+## Installation
+
+The easiest way to install is by downloading the latest PHAR and putting it into your path. For example:
+
+```
+curl -O -L https://github.com/mglaman/drupal-check/releases/download/1.0.5/drupal-check.phar
+mv drupal-check.phar /usr/local/bin/drupal-check
+```
 
-For the best experience with this tool is best used against a vanilla Drupal project. It will work against custom Drupal 
-projects, but there can be conflicts against other development tools added to projects.
+### Composer
 
-Check deprecations
+You can also install this globally using composer like so:
 
 ```
-# Example: Against address contrib
-drupal-check /path/to/drupal8/modules/contrib/address
-drupal-check -d /path/to/drupal8/modules/contrib/address
+composer global require mglaman/drupal-check
 ```
 
-Check static analysis
+Refer to Composer's documentation on how to ensure global binaries are in your PATH: https://getcomposer.org/doc/00-intro.md#manual-installation.
+
+Note: you can also install this locally to your project and run it from that project's composer bin directory.
+
+### Build From Source
+
+You can also clone this repository and build the PHAR from source:
+
+First, install humbug/box:
 
 ```
-# Example: Against address contrib
-drupal-check -a /path/to/drupal8/modules/contrib/address
+composer global require humbug/box
 ```
 
-Check static analysis and deprecations
+Next, clone the repo and build:
 
 ```
-# Example: Against address contrib
-drupal-check -ad /path/to/drupal8/modules/contrib/address
+git clone https://github.com/mglaman/drupal-check.git
+cd drupal-check
+composer install --prefer-dist
+box -v
 ```
 
-Coming soon: code style w/ phpcs integration.
+## Usage
 
-## Install
+This tool works on contrib modules within a Drupal repository.
 
-Download the latest Phar from https://github.com/mglaman/drupal-check/releases/latest, move into your path, profit!
+### 1. cd into a Drupal Directory
 
-Or, something like:
+You can run this tool within any Drupal project. But, for best results, create a fresh Drupal directory on the latest Drupal:
 
 ```
-curl -O -L https://github.com/mglaman/drupal-check/releases/download/1.0.5/drupal-check.phar
-chmod +x $(pwd)/drupal-check.phar
-ln -s $(pwd)/drupal-check.phar /usr/local/bin/drupal-check
-drupal-check /path/to/drupal/code_to_analyze
+composer create-project drupal-composer/drupal-project:8.x-dev drupal --no-interaction --stability=dev
+cd drupal
 ```
 
-## Building
+### 2. Run drupal-check
 
-The phar is built using [humbug/box](https://github.com/humbug/box)
+Usage:
 
-macOS with bew:
+  ```
+  drupal-check [OPTIONS] [MODULE]
+  ```
+
+Options:
+
+* `-a` Check analysis
+* `-d` Check deprecations
+
+Examples:
+
+* Check the address contrib module: `drupal-check web/modules/contrib/address`
```

Comment:
Should the examples have inline code, or use three ticks so it's a newline block?